src/kontrol/prove.py

```diff
@@ -192,11 +235,13 @@ def _run_cfg_group(
     trace_rewrites: bool,
     auto_abstract_gas: bool,
     port: int | None,
+    run_constructor: bool = False,
 ) -> dict[tuple[str, int], tuple[bool, list[str] | None]]:
+    print(f'run_constructor={run_constructor}')
```

Comment:
Just want to confirm: is it intentional or left over from debugging?

Reply:
Removed.